mutiny-core/src/logging.rs

```diff
@@ -1,9 +1,92 @@
+use std::sync::{
+    atomic::{AtomicBool, Ordering},
+    Arc, Mutex,
```

Comment:
this is using the syncronous mutex, shouldn't it use the `crate::utils::Mutex`?

Reply:
there's so many `Mutex`'s I never know which one to use